### ISSUE TITLE
duplicated exports and duplicated import names

### ISSUE THREAD

**Issue by GerardRodes** [rollup/rollup#3583]
- Rollup Version: 2.10.7
- Operating System (or Browser): Ubuntu 18.04
- Node Version (if applicable): v12.16.2
- Link to reproduction (IMPORTANT, read below): https://rollupjs.org/repl/?version=2.10.7&shareable=JTdCJTIybW9kdWxlcyUyMiUzQSU1QiU3QiUyMm5hbWUlMjIlM0ElMjJtYWluLmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmV4cG9ydCUyMCU3QiUyMGRlZmF1bHQlMjBhcyUyMENvbXBvbmVudF9vbmUlMjAlN0QlMjBmcm9tJTIwJy4lMkZDb21wb25lbnRfb25lLmpzJyU1Q24lMjIlMkMlMjJpc0VudHJ5JTIyJTNBdHJ1ZSU3RCUyQyU3QiUyMm5hbWUlMjIlM0ElMjJDb21wb25lbnRfb25lLmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmltcG9ydCUyMColMjBhcyUyMGljb25zJTIwZnJvbSUyMCcuJTJGaWNvbnMlMkZpbmRleCclNUNuY29uc3QlMjBfX2NvbXBvbmVudF9fJTIwJTNEJTIwJTdCJTIwaWNvbnMlMjAlN0QlNUNuJTVDbmV4cG9ydCUyMGRlZmF1bHQlMjBfX2NvbXBvbmVudF9fJTIyJTJDJTIyaXNFbnRyeSUyMiUzQWZhbHNlJTdEJTJDJTdCJTIybmFtZSUyMiUzQSUyMmljb25zJTJGb25lLmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmNvbnN0JTIwX19pY29uX18lMjAlM0QlMjAlN0IlN0QlNUNuJTVDbmV4cG9ydCUyMGRlZmF1bHQlMjBfX2ljb25fXyUyMiUyQyUyMmlzRW50cnklMjIlM0F0cnVlJTdEJTJDJTdCJTIybmFtZSUyMiUzQSUyMmljb25zJTJGdHdvLmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmNvbnN0JTIwX19pY29uX18lMjAlM0QlMjAlN0IlN0QlNUNuJTVDbmV4cG9ydCUyMGRlZmF1bHQlMjBfX2ljb25fXyUyMiUyQyUyMmlzRW50cnklMjIlM0FmYWxzZSU3RCUyQyU3QiUyMm5hbWUlMjIlM0ElMjJpY29ucyUyRmluZGV4LmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmV4cG9ydCUyMCU3QiUyMGRlZmF1bHQlMjBhcyUyMG9uZSUyMCU3RCUyMGZyb20lMjAnLiUyRm9uZSclNUNuZXhwb3J0JTIwJTdCJTIwZGVmYXVsdCUyMGFzJTIwdHdvJTIwJTdEJTIwZnJvbSUyMCcuJTJGdHdvJyUyMiUyQyUyMmlzRW50cnklMjIlM0FmYWxzZSU3RCU1RCUyQyUyMm9wdGlvbnMlMjIlM0ElN0IlMjJmb3JtYXQlMjIlM0ElMjJlcyUyMiUyQyUyMm5hbWUlMjIlM0ElMjJteUJ1bmRsZSUyMiUyQyUyMmFtZCUyMiUzQSU3QiUyMmlkJTIyJTNBJTIyJTIyJTdEJTJDJTIyZ2xvYmFscyUyMiUzQSU3QiU3RCU3RCUyQyUyMmV4YW1wbGUlMjIlM0FudWxsJTdE

### Expected Behavior
```js
const __icon__ = {};

export { __icon__ as _ };
```
```js
import { _ as __icon__$1 } from './one-b745e937.js';
```

### Actual Behavior
```js
const __icon__ = {};

export { __icon__ as _, __icon__ as a };
```
```js
import { _ as __icon__$1, a as __icon__$1 } from './one-b745e937.js';
```

**Comment by TrySound** [comment]
This looks similar to https://github.com/rollup/plugins/issues/392

**Comment by lukastaegert** [comment]
Thanks for spotting, this will be fixed in #3590 
https://rollupjs.org/repl/?circleci=11323&shareable=JTdCJTIybW9kdWxlcyUyMiUzQSU1QiU3QiUyMm5hbWUlMjIlM0ElMjJtYWluLmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmV4cG9ydCUyMCU3QiUyMGRlZmF1bHQlMjBhcyUyMENvbXBvbmVudF9vbmUlMjAlN0QlMjBmcm9tJTIwJy4lMkZDb21wb25lbnRfb25lLmpzJyU1Q24lMjIlMkMlMjJpc0VudHJ5JTIyJTNBdHJ1ZSU3RCUyQyU3QiUyMm5hbWUlMjIlM0ElMjJDb21wb25lbnRfb25lLmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmltcG9ydCUyMColMjBhcyUyMGljb25zJTIwZnJvbSUyMCcuJTJGaWNvbnMlMkZpbmRleCclNUNuY29uc3QlMjBfX2NvbXBvbmVudF9fJTIwJTNEJTIwJTdCJTIwaWNvbnMlMjAlN0QlNUNuJTVDbmV4cG9ydCUyMGRlZmF1bHQlMjBfX2NvbXBvbmVudF9fJTIyJTJDJTIyaXNFbnRyeSUyMiUzQWZhbHNlJTdEJTJDJTdCJTIybmFtZSUyMiUzQSUyMmljb25zJTJGb25lLmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmNvbnN0JTIwX19pY29uX18lMjAlM0QlMjAlN0IlN0QlNUNuJTVDbmV4cG9ydCUyMGRlZmF1bHQlMjBfX2ljb25fXyUyMiUyQyUyMmlzRW50cnklMjIlM0F0cnVlJTdEJTJDJTdCJTIybmFtZSUyMiUzQSUyMmljb25zJTJGdHdvLmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmNvbnN0JTIwX19pY29uX18lMjAlM0QlMjAlN0IlN0QlNUNuJTVDbmV4cG9ydCUyMGRlZmF1bHQlMjBfX2ljb25fXyUyMiUyQyUyMmlzRW50cnklMjIlM0FmYWxzZSU3RCUyQyU3QiUyMm5hbWUlMjIlM0ElMjJpY29ucyUyRmluZGV4LmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmV4cG9ydCUyMCU3QiUyMGRlZmF1bHQlMjBhcyUyMG9uZSUyMCU3RCUyMGZyb20lMjAnLiUyRm9uZSclNUNuZXhwb3J0JTIwJTdCJTIwZGVmYXVsdCUyMGFzJTIwdHdvJTIwJTdEJTIwZnJvbSUyMCcuJTJGdHdvJyUyMiUyQyUyMmlzRW50cnklMjIlM0FmYWxzZSU3RCU1RCUyQyUyMm9wdGlvbnMlMjIlM0ElN0IlMjJmb3JtYXQlMjIlM0ElMjJlcyUyMiUyQyUyMm5hbWUlMjIlM0ElMjJteUJ1bmRsZSUyMiUyQyUyMmFtZCUyMiUzQSU3QiUyMmlkJTIyJTNBJTIyJTIyJTdEJTJDJTIyZ2xvYmFscyUyMiUzQSU3QiU3RCU3RCUyQyUyMmV4YW1wbGUlMjIlM0FudWxsJTdE
